README.md

```diff
@@ -19,4 +19,4 @@ Instructions for installing packages and running virtual environment
 4) Ensure that the run.sh file has executable permissions by typing 
```

Comment:
Can you please add this update to 2) in README?
'virtualenv -p /bin/python3 venv' change to 'virtualenv -p /bin/python3.6 venv'

I would update myself since it was my fault but I hardly think an entire PR is needed for a typo.